src/App.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import Cell from './Cell';
+
+class App extends Component {
```

Comment:
***Code Quality: NitPick***
Wont this make sense if it's a functional component